recipes/cppcheck/all/patches/0001-cmake-source-dir-2.5.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/tools/CMakeLists.txt b/tools/CMakeLists.txt
```

Comment:
It would be nice if this patch could be upstreamed.
The patch looks small and clean.